.env.sample

```diff
@@ -40,4 +44,59 @@ TELEMETRY_ON_OFF = ON/OFF
 
 TELEMETRY_TOPIC = {{env_name}}.telemetry.raw                                      
 VALIDATE_ENTITIES = "ON/OFF"                                                    // validate entities enable and disable
-PROGRAM_JOIN_ON_OFF = "ON/OFF"                                                  // Program Join Feature enable and disable
\ No newline at end of file
+PROGRAM_JOIN_ON_OFF = "ON/OFF"                                                  // Program Join Feature enable and disable
+
+
+# creation portal migration Script
+
+MASTER_USER_EMAIL = "n11@yopmail.com"                                            // master user name of the sunbird enviornment to generate token
+MASTER_USER_PWD = "password"                                                     // master user pwd of the sunbird enviornment to generate token
+TOKEN_GEN_GRANT_TYPE = "password-grant"                                          // grant type to generate token for the sunbird enviornment
+TOKEN_GEN_CLIENT = "client-id"                                                   // client to generate token for the sunbird enviornment
+TOKEN_GEN_CLIENT_SECRET = "client-secret-key"                                    // client secret to generate token for the sunbird enviornment
+
+ED_BASE_URL = "https://dev.sunbirded.org/"                                       // base host url of the sunbird enviornment
+ED_AUTHORIZATION = "Bearer + 'token'"                                            // base bearer token with all the permissions to run the script of the sunbird enviornment
+
+CREATION_PORTAL_BASE_URL = "https://dock.sunbirded.org/"	                             // creation portal Url
+
+CREATION_PORTAL_AUTHORIZATION = "Bearer + 'token'"	                             // creation portal bearer token with all the permissions  for below urls, 
+
+
+# Migration
+
+    # API'S required for migration
+
+        GEN_TOKEN: "auth/realms/sunbird/protocol/openid-connect/token"                 // To generate token
```

Comment:
@jayesh9747 format the env sample no additional space is required in the left side